Tools/SetupUnityPackage/UnityUtils/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Reflection;
+using UnityEngine.Scripting;
+using System.Runtime.CompilerServices;
+
+// Information about this assembly is defined by the following attributes. 
+// Change them to the values specific to your project.
+
+[assembly: AssemblyTitle("UnityUtils")]
+[assembly: AssemblyDescription("Utils to support the Realm SDK in Unity projects")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Realm")]
+[assembly: AssemblyProduct("")]
+[assembly: AssemblyCopyright("")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
```

Comment:
These will fail the build because assembly information is automatically generated on build. It's a good idea to add them, but they should be specified in the csproj instead.